### PR TITLE
Bump deCONZ to 2.05.81

### PR DESCRIPTION
**Pull request by pattyland**
Bump deCONZ to 2.05.81

Release notes: https://github.com/dresden-elektronik/deconz-rest-plugin/tree/V2_05_81_stable

### CODE REVIEW

**Comment by frenck** [comment]
I think we should not.

A) As we decided before with 2.05.80 (beta of this release), it removed the possibility to update. As the release notes of clearly state only manual updates work at this point:

image

There was a fix: https://github.com/dresden-elektronik/deconz-rest-plugin/pull/3256
But considering the warning, we can't tell its enough.

Lastly, 2.05.82 is already tagged, but not released yet. Apparently a hotfix is inbound as well.

**Comment by fi-sch** [comment]
@frenck 
That's cleared out in this issue that I've created in DE repository: dresden-elektronik/deconz-rest-plugin#3266
Manual update requirement isn't the case with the latest deCONZ.

**Review by fi-sch — CHANGES_REQUESTED** [review]
Shouldn't the value be set to "2.05.81"?

File: deconz/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "deCONZ",
-  "version": "6.2.3",
+  "version": "6.2.4",
```
Comment (pvizeli) [inline deconz/config.json:3]:
```suggestion
  "version": "6.3.0",
```

File: deconz/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 6.2.4
```
Comment (pvizeli) [inline deconz/CHANGELOG.md:3]:
```suggestion
## 6.3.0
```

**Comment by pvizeli** [comment]
@pattyland would be nice if you can test it next time...

So we have the issue now, aarch64 have no x.81

**Comment by pvizeli** [comment]
https://twitter.com/pvizeli/status/1305502465027190784

**Comment by manup** [comment]
Good hint, yes but will take a bit, the aarch64 needs to be integrated into the normal stable/beta builds.
Should be ready till Friday.

**Comment by manup** [comment]
Note however that the 2.5.82 (beta) is already available also for aarch64, compared to 2.5.81 there are some new device support related changes. https://github.com/dresden-elektronik/deconz-rest-plugin/releases/tag/V2_05_82_1

http://deconz.dresden-elektronik.de/raspbian/alpha/deconz-dev_2.05.82-debian-stretch-beta_arm64.deb

**Comment by manup** [comment]
Clarification for the firmware update, since .81 the update via REST-API and Phoscon App is enabled again with the same firmware which was part of .79 release. The release fixes some bugs which people had while updating the firmware.

The new firmware mentioned above, is too young to roll out "automatically" yet, hence until battle tested it remains to be installed manually, likely a candidate for the next release.

**Comment by pvizeli** [comment]
@manup thanks for you hard work. What do you like that we use should use, stable or beta for long term? Do you have plans to make more stable releases (sure stable is a hard word :) ) because of device updates? Since people make also issues on your site, I will give you a voice for what we should distribute to the peoples.

**Comment by manup** [comment]
@pvizeli Thanks :) I think the safest approach is to use the stable versions. We are aiming to to make a stable version 2–3 weeks after each beta version. The beta versions are scheduled for the 15th of each month and may include larger patches and new device support PR's. Based on the beta the stable versions are mainly focused on fixing bugs.

In that hindsight stable will get new devices, with a delay of roughly a month behind beta.

I've cleaned up the aarch64 uploads into new `/debian/{stable | beta}/` folders. The beta was formerly uploaded to `/raspbian/alpha/` where it doesn't belong, the files are still available there to not break downloads, but will eventually be removed in a few months.

http://deconz.dresden-elektronik.de/debian/beta/deconz_2.05.82-debian-stretch-beta_arm64.deb

http://deconz.dresden-elektronik.de/debian/stable/deconz_2.05.81-debian-stretch-stable_arm64.deb

**Comment by dedepene** [comment]
I hate to be "that guy" but is there any ETA for merging this and making it widely available? I have a little project that hangs in the air, waiting for the support of devices the latest Deconz brings...thanks!

**Comment by frenck** [comment]
@dedepene The answer is always: No, there is no ETA.

**Comment by fi-sch** [comment]
Is there any chance of bumping this to the latest stable version anytime soon?

**Comment by SilverWx** [comment]
@fi-sch 
Do not ask for ETA please

**Comment by fi-sch** [comment]
I'm not asking for ETA. This is a simple question, any answer would be appreciated (yes/no). I just wanted to know if this could possibly take two months or two weeks. No pressure from my side.

I know that it's not a commercial product and a lot of volunteer work is going on. I do respect and appreciate that also.

**Comment by stigvi** [comment]
> @fi-sch
> Do not ask for ETA please

But it would be nice to have a release based on a stable version instead of a beta/pre-release as we have now as the current version. I don't know about others, but I have to restart HA almost every day because of websocket problems with the deconz add-on. It got a lot better with version 0.115, but I would very much have a version with release notes like "This stable release contains mainly bug fixes and robustness improvements."

**Pull request merged**